src/http_server/cargo_dash_http_server.py

```diff
@@ -0,0 +1,124 @@
+import asyncio
+from aiohttp import web
+
+routes = web.RouteTableDef()
+
+
+@routes.get('/allnodes')
+async def get_all_nodes(request):
+    ''' Receives a get request that asks for all nodes.
+
+        Returns array with all nodes in json format.
+
+        Ex. returns
+        [{
+            type : 'DistanceNode',
+            purpose : 'Front view object distance',
+            variables : {
+                ...
+            }
+        },
+        {
+            type : 'SteeringNode',
+            purpose : 'Steering wheel',
+            variables : {
+                ...
+            }
+        },
+            ...
+        ]
+    '''
+    pass
+
+
+@routes.get('/node/{id}')
+async def get_node(request):
+    ''' Receives a get request that asks for a specific node with a certain id.
```

Comment:
Ik vind dit een beetje vaag omschreven
Bedoel je zoiets?
```suggestion
    ''' Returns node information based on node ID
```